README.md

```diff
@@ -83,6 +135,7 @@ In the final state it should look something like this:
       [buttonCaption]="'Select Images!'"
       [dropBoxMessage]="'Drop your images here!'"
       [extensions]="['jpg','png','gif']"
+      [class]="'customClass'"      
```

Comment:
Can add `[style]` directive but it will mislead user. Good to apply either `[class]` or `[style]` because `[style]` overrides `[class]`. Do you still want me to add? @aberezkin